### DISCUSSION TITLE
ITN Contest

### DISCUSSION THREAD

**Discussion by 0xsingular** [teaxyz/white-paper#113] [Ideas]
As far as we know the ITN will be launched in the near future, for a better understanding of the protocol and how it works it would be good to launch a contest in the context of the ITN.
For example, we know that the staking ratio differs for different packages, so the contest could be based on this aspect, so that users do further research to understand how the protocol works and stake TEA tokens in favour of the package that has a higher teaRank or staking ratio, so they get more incentives.

The purpose of this contest is to incentivize users to make some extra effort in understanding how the protocol works. (Not just to follow the steps indicated in the guide, 1, 2, 3, ...).

This way users with the knowledge gained will have a better understanding of the tea protocol, and will apply this knowledge on mainnet, as it is important in the early stages of mainnet launch for users with a little more knowledge to participate in maintaining the protocol.

**Comment by thomas-borrel** [comment]
Hi @0xsingular,
I completely agree with you. We'll publish more on this in the coming weeks (although there are some hints [there](https://tea.xyz/blog/10-ways-for-non-devs-to-participate-in-itn)), but here's a sneak preview.
ITN will be split into multiple challenges with each challenge containing multiple quests. Each quest will require specific actions to be performed - some simple, like creating an account, some more complex/involved, like experimentation with staking, reporting vulnerabilities, etc.
The current plan includes over 30 quests so all users can get a better understanding of how the **_tea_** Protocol functions and be ready for mainnet. Each quest has a number of points associated - These points are automatically assigned and accumulated by participating in various Incentivized Testnet activities and may become convertible into claimable rewards at a future date.
These quests will also help the core team understand how users are interacting with the protocol and where it could be improved/optimized. That way everyone can learn from each other and make the protocol stronger and more impactful.

Thoughts?

**Comment by 0xsingular** [comment]
Great! I was referring to activities that have a point of variability that will prompt testnet participants to do their own research and be involved throughout the testnet.
Let it not be that they completed all the activities in one day just for points.
However I have previously read the ways to participate in the testnet and there are activities that require more involvement.

I like everything that is included in the testnet, and it seems that the team has also thought about these aspects. I look forward to the testnet.
P.S. My favorite part of the protocol is that each OSS project will be its own DAO, this will allow for a more positive and constructive community/contributor connection with the teams/devs in the context of the other benefits/tools offered by the tea protocol. P.S.S: My opinion.

**Reply by thomas-borrel** [reply]
Great! How specifically do you envision the OSS project DAO allowing for a more positive and constructive community/contributor connection with the maintainers? Are you talking about projects being able to issue bounties? Recruit external expertise to augment specific parts of their project? Something totally different?

**Reply by 0xsingular** [reply]
Since the tea protocol provides all the necessary tools for project-community (contributor) collaboration, and provides relative financial support (TEA rewards), it is possible to maximize the effectiveness of these tools for projects.

The idea is that projects based on these tools can form an incubation environment by working with the community at all stages of development/growth.
For example: as a project I would establish and present a long and short term plan (updated for each quarter), with the stages of growth and the forms in which the community can contribute plus establish the benefits for the contributors, direct or indirect (e.g. if it is an infrastructure/tool project then the contributors may receive some benefits from the projects offering an end product built on these tools).
The forms of contribution may be different depending on the specifics of the project, but in general:
1. Marketing (social engagement, content, translations
2. Testing
3. Financial support (donations) for direct project needs or for entities undertaking specific tasks for the project.
4. Communication and collaboration (establishment of focus groups)

In this way the community/contributors will have a better understanding of the project and the effect of direct contribution (building) will lead the community to use the project products and provide support over a long period of time.


The points idea that has now gained momentum and shows community engagement and understanding of the possible benefits, mostly financial, has gained momentum only because it provides clarity about the possible benefits, but for the most part it is expected that after these benefits are achieved the community will not engage in these projects because there are other projects that provide points).
So we have (a.) projects where the team has built something in the background, just making some announcements of what they are going to launch (which is largely unclear to the community and what the community has not been able to verify) and offered points for social engagement, after which some financial benefits.
Now (b.), OSS project registered under tea protocol, which worked with the community, presented detailed long/short term plans, a diversification of forms of direct contribution, POSSIBILITY TO VERIFY IN A SIMPLE FORM (teaRank) THE RESULTS/TECHNICAL DEVELOPMENT OF THE PROJECT, and also clearly set out the benefits to contributors.

A pioneer project is needed that will choose this path for an expansion of this idea.

Or it is possible to create a society (group/community) not directly associated with tea, but whose main purpose/values/vision, constitution and framework will be focused on contributing to OSS projects through the tea protocol.

If I were a developer I would also want to see the rank of each contributor within the tea protocol, so for more specific tasks (contributions) I would involve more trusted contributors. I opened [a new discussion on the idea of creating contributor rank](https://github.com/teaxyz/white-paper/discussions/114#discussion-6022258).

**Reply by thomas-borrel** [reply]
Thank you @0xsingular - Very interesting. For the package-focused DAO, I think it's important to create communities that are not directly associated with tea but rather focus on supporting the need of the specific package. Interesting thought with each project sharing a roadmap and what their plans are to attract more stakers and other contributors. Good for all to see what's being planned, but that also sets expectations which makes it a double-edged sword.
